### ISSUE TITLE
Redirect Plan

### ISSUE THREAD

**Issue by blainehussey** [SolaceDev/solace-dev-tutorials#37]
- [x] Create a redirect plan in Excel for all existing Samples URLs (approved by @Mrc0113)
- [ ] Configure netlify.toml files for redirects
- [x] Add redirect for [solace.dev/tutorials](https:/solace.dev/tutorials) --> [tutorials.solace.dev](https://tutorials.solace.dev)
- [x] Add redirects for old samples hosted on solace.com on WPEngine and update spreadsheet
- [x] Remove and backup old solace.com/samples/* folders
- [x] Delete the jekyll docs on the older samples
- [x] Update older samples readme to link to tutorials

**Comment by blainehussey** [comment]
@TamimiGitHub - I'm going to get rolling on the redirects under solace.com, I see there's a few other items in the list. Just wanted to confirm I'm good to move forward with this? Thanks Brother.

**Comment by TamimiGitHub** [comment]
Hey @blainehussey let's go ahead with the redirects please 👍 
@Mrc0113 FYI

**Comment by blainehussey** [comment]
@Mrc0113 & @TamimiGitHub - Redirects on Solace.com are all shipped and have been tested (spreadsheet updated). I will go ahead and move forward with deleting the `/samples/*` folders on Solace.com as they're no longer needed. Do you want me to do a backup and send it as a ZIP?

I guess the next steps would be to remove the `_docs` folders in the repos, I will leave that for you guys.

**Comment by TamimiGitHub** [comment]
Thanks for the update Blaine. Just to confirm, is the `feature_*` still in the redirects or have they been removed? I am fine with  removing the samples dir and rerouting solace.com/samples/ --> tutorials.solace.dev/

**Comment by blainehussey** [comment]
The `feature` comment I mentioned in Slack was more just an observation, it's definitely something you and I should probably update while we're making other updates in Tutorials but I didn't take action on any file name and/or path updates, I only focused on the redirects for `solace.com/samples/*` to `tutorials.solace.dev/*`.

**Comment by TamimiGitHub** [comment]
I made the changes in the live site and update the spreadsheet. Would you be able to confirm the redirects just for those ones that had features in them are corrected? Thanks!

**Comment by blainehussey** [comment]
As discussed, I'm going to action the redirects for those links you updated in the spreadsheet today. Thanks for fixing those and pushing those updates @TamimiGitHub

**Comment by blainehussey** [comment]
Hey @TamimiGitHub,
can you please test/review those new redirects when you have time, please note you may need to clear your browser cache if you had tried testing them yesterday, I ran into this problem. I've tested them and they should be good but if you find any that are giving you problems let me know and I'll investigate.

Thanks again.

**Comment by TamimiGitHub** [comment]
Sounds good thanks blaine! i’ll take a look 🙏

**Comment by TamimiGitHub** [comment]
Deleted the jekyll docs and updated the readme of all samples.

**Comment by TamimiGitHub** [comment]
@blainehussey can you please check if there is anything left for this issue? Otherwise we can go ahead and close it!

**Comment by blainehussey** [comment]
Yeah, I'm going to go ahead and close this issue @TamimiGitHub, all looks good on our end. In doing some final searching, I did find a page on the Docs site that contains old Samples URLs, so I have sent an email off (cc'd both you and @Mrc0113) to the Docs team to update this. I have also requested an entire search of the Docs site for references to the old Samples. I realize that we have redirects in place but old links should still be updated. On a plus side, it appears Google is already reindexing the new Tutorials pages.

@Mrc0113 - I noticed [this page on the Community](https://solace.community/discussion/575/replay-with-solace-learn-more-a-few-ways) has references to **Java**, **C**, and **JavaScript**, is there anyway to update this post with the new URLs?

Thanks Team, great work on all of this.
